sky/utils/command_runner.py

```diff
@@ -29,6 +29,8 @@
 # Note that "-" is mandatory for rsync and means all patterns in the ignore
 # files are treated as *exclude* patterns.  Non-exclude patterns, e.g., "!
 # do_not_exclude" doesn't work, even though git allows it.
+# TODO(cooperc): Avoid using this, and prefer utils in storage_utils instead for
+# consistency between bucket upload and rsync.
```

Comment:
Please remember to file an issue for this

Reply:
Also, just wondering, if we should align the behavior for rsync with the bucket upload, so we can update our doc for the behavior of gitignore.

Reply:
Yes, that's what I mean to do for this TODO.

Reply:
#5006